kedro/template/{{ cookiecutter.repo_name }}/kedro_cli.py

```diff
@@ -249,6 +253,11 @@ def build_docs():
         ]
     )
     call(["sphinx-build", "-M", "html", "docs/source", "docs/build", "-a"])
+    if open_docs:
+        p = Path("docs/build/html/index.html").resolve().as_uri()
```

Comment:
Given the difference in path separators, I suspect this will not play nice with Windows. How about going for something like this instead:
```(Path.cwd() / "docs" / "build" / "html" / "index.html").as_uri()```
and let `pathlib` handle the separators?
This might also help you get rid of the mock object in your test, which is nice.

Reply:
Ah, that's nice. Thanks, should be sorted now. :)